Add files for Giggity support.

TheTincho opened the pull request:
As discussed in prometheus/promcon#28, this adds the necessary files for the Giggity app to provide the conference schedule in Android devices.

There are two pending things:
* Adding the proper name for the rooms, as I had no idea about them.
* Checking if the automatic build will just copy these files verbatim. I did not have the tooling locally to test that.

juliusv commented on content/2017-munich/giggity.json:
```diff
@@ -0,0 +1,34 @@
+{
+    "version": 2017081200,
+    "url": "https://tincho.org/~tincho/schedule.xml",
```
This would be https://promcon.io/2017-munich/schedule.xml I guess?

TheTincho replied:
Oops. Yes, sorry :)

juliusv commented:
The site compilation fails with errors about the json and xml file not having any compilation rules.

It works when you add this to the top-level `Rules` file:

```
compile '/**/*.json' do
  write item.identifier
end 

compile '/**/*.xml' do
  write item.identifier
end
```

That's probably not the right way to do things, but we have no time to do things right, so it's fine :)

juliusv commented:
The room name "Main room" is fine, since we'll only have one room anyways.

TheTincho commented:
I fixed the compilation errors (this time I installed all the ruby tools, so I could verify it works), and updated the schedule to match today's changes. PTAL?

juliusv commented:
Thanks!

TheTincho commented:
I think I should add a link to the app in the schedule, will send another PR
